.github/workflows/changelog.yml

```diff
@@ -0,0 +1,63 @@
+name: Update Changelog
+
+on:
+  workflow_dispatch:
+
+# only run at most one instances of this workflow at a time for each branch
+# resp. tag. Starting a new one cancels previously running ones.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  update-changelog:
+    runs-on: ubuntu-22.04
+
+    steps:
+    # Checkout the repository
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+
+    # Run the update script
+    - name: Setup venv and install requests
+      run: |
+        pip install requests
+
+    - name: Setup gh utility
+      run: |
+        (type -p wget >/dev/null || (sudo apt update && sudo apt-get install wget -y))
+        sudo mkdir -p -m 755 /etc/apt/keyrings
+        out=$(mktemp) && wget -nv -O$out https://cli.github.com/packages/githubcli-archive-keyring.gpg
+        cat $out | sudo tee /etc/apt/keyrings/githubcli-archive-keyring.gpg
+        sudo chmod go+r /etc/apt/keyrings/githubcli-archive-keyring.gpg
+        echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+        sudo apt update
+        sudo apt install gh -y
+
+    - name: Run python script
+      env:
+        GH_TOKEN: ${{ github.token }}
+      run: |
+        cd dev/releases
+        python3 release_notes.py $(gh release list | grep Latest| sed 's/\s.*$//' | cut -c 2- | awk -F. -v OFS=. '{$NF += 1 ; print}')
```

Comment:
This should be cleaned up in a follow-up: the release_notes.py script should just query for the latest release if given no argument. And then use `gh release list --json name,isLatest --jq SOMEQUERY` to only get the latest release (or just omit the query and do it in python -- parse the json produced by that command and take the with isLatest set to true